applications/GeoMechanicsApplication/tests/lysmer_boundary_stiff_column2d_quad/README.md

```diff
@@ -0,0 +1,31 @@
+# Lysmer absorbing boundary conditions on a column made of rectangulars
+
+This test verifies the implementation of the Lysmer absorbing boundary conditions. 
+
+## Setup
+
+The test is performed for an 1D column that is made with 2D rectangular elements. The column height and width are 10,000 mm and 1,000 m, respectively.
```

Comment:
Very minor, but I think this should be mm?
```suggestion
The test is performed for an 1D column that is made with 2D rectangular elements. The column height and width are 10,000 mm and 1,000 mm, respectively.
```

Reply:
Thank you. It was a typo.